internal/serialization/object_data_test.go

```diff
@@ -510,8 +492,8 @@ func TestObjectDataInput_ReadingAfterError(t *testing.T) {
 	assert.Error(t, i.Error())
 	i.ReadObject()
 	assert.Error(t, i.Error())
-	i.ReadData()
-	assert.Error(t, i.Error())
+	//i.ReadData()
```

Comment:
nit: this and the following lines look like a leftover

Reply:
I've removed WriteData/ReadData functions but didn't remove the tests. They should be gone now: https://github.com/hazelcast/hazelcast-go-client/pull/555/commits/f8335e3a66165c8383ce93ed08fc6127f3af1375